eventnotifier/slack/slack.go

```diff
@@ -178,12 +178,10 @@ func (n *Notifier) notifyForDisruptionCron(disruptionCron *v1beta1.DisruptionCro
 	return nil
 }
 
-func (n *Notifier) sendMessageToUserChannel(obj client.Object, slackMsg slackMessage, logger *zap.SugaredLogger) error {
-	objKind := obj.GetObjectKind().GroupVersionKind().Kind
-
+func (n *Notifier) sendMessageToUserChannel(slackMsg slackMessage, logger *zap.SugaredLogger) error {
 	emailAddr, err := mail.ParseAddress(slackMsg.UserInfo.Username)
 	if err != nil {
-		logger.Warnw("invalid user info email in %s %s: %w", objKind, obj.GetName(), err)
+		logger.Warnw("username could not be parsed as an email address", "err", err, "username", slackMsg.UserInfo.Username)
```

Comment:
This could be at Info, it's frequently expected to happen

Reply:
Info makes sense yes!